concordium-consensus/src/Concordium/External/GRPC2.hs

```diff
@@ -34,6 +34,7 @@ import qualified Concordium.GlobalState.ContractStateV1 as StateV1
 import Concordium.ID.Types
 import qualified Concordium.Queries as Q
 import Concordium.Types
+import qualified Concordium.Types.Queries as TQ
```

Comment:
Same comment.

Reply:
I followed the `as Q` for other Queries we import. But since you commented it is now Queries.